start/sw/web-app/django.md

```diff
@@ -10,130 +10,103 @@ Django is a high-level Python Web framework that encourages rapid development an
 Django is one of the most popular Python Web Frameworks. It has very useful web page. Visit it by clicking on [this link](https://www.djangoproject.com/). 
 Because you will need to work with Python packages, write this command to your command line.
 
-For Python:
-
-```
-sudo dnf install python-pip
-```
-
-or Python 3:
-
 ```
-sudo dnf install python3-pip
+$ sudo dnf install python3-pip
 ```
 
-# Get started
+## Get started
 
-## How to install virtualenv
+### How to install virtualenv
 
 It's recommended to keep your project inside virtual environment. But you can skip this step if you want. 
 Enter this command to your command line for installation:
 
 ```
-sudo dnf install python-virtualenv
-```
-
-or installation for Python3:
-
-```
-sudo dnf install python3-virtualenv
+$ sudo dnf install python3-virtualenv
 ```
 
 
 Get inside your working directory or create new where you want to store your virtual environment:
 
 ```
-mkdir working-dir
-cd working-dir
+$ mkdir working-dir
+$ cd working-dir
 ```
 
 Now we will create virtual environment called venv inside this directory using following command:
 
 ```
-virtualenv venv
+$ virtualenv venv
 ```
 
 When it's done, we will activate our virtualenv. Now we will use following commands:
 
 ```
-cd venv
-source bin/activate
+$ cd venv
+$ source bin/activate
 ```
 
 When it's activated we will see indicator of same name as our virtualenv. 
 Now we will check if we really have empty environment:
 
 ```
-pip freeze
+$ pip freeze
 ```
 
 It will return nothing.
 
-## How to install Django
+### How to install Django
 
 Now if you installed `virtualenv` do not leave it, we will install Django now. If you installed `virtualenv` use this command 
 inside your working directory with activated `virtualenv`:
 
 ```
-pip install django
+$ pip3 install django
```

Comment:
I think this should be on TOP of this page, as this is actually what user is looking for (not virtualenv), so maybe just mention this is to be done in already created virtualenv?

It should not be a complete guide, but rather a quickstart or reference.

Reply:
If this ever gets mentioned without invoking virtualenv, make sure it is invoked with `--user` otherwise it will die on permission error and users tend to "solve" such problems with `sudo`.

Reply:
Yes, having enabled virtualenv is pretty much what the previous text says (L60). But, maybe it's worth mentioning you can use `--user` instead, if it's a reasonable alternative.